addons/base_setup/models/res_users.py

```diff
@@ -12,6 +13,9 @@ class ResUsers(models.Model):
     def web_create_users(self, emails):
         emails_normalized = [tools.mail.parse_contact_from_email(email)[1] for email in emails]
 
+        if not hasattr(self, 'email_normalized'):
```

Comment:
`hasattr` returns True for method too, it is preferable to use `self._fields` instead (it is more readable too).
```suggestion
        if 'email_normalized' not in self._fields:
```

Reply:
done, thank's :heavy_check_mark: